### PR TITLE
LL-6087 add tracking to view in explorer

### PR DESCRIPTION
**Pull request by valpinkman**
add tracking to View in Explorer action

### Type

improvement

### Context

LL-6087

### Parts of the app affected / Test plan

view in explorer actions now trigger a tracking

### CODE REVIEW

File: src/screens/OperationDetails/Footer.js [hunk 1]
```diff
@@ -34,7 +46,7 @@ function Footer({ url, urlWhatIsThis }: Props) {
           event="OperationDetailViewInExplorer"
           type="primary"
           title={<Trans i18nKey="operationDetails.viewInExplorer" />}
-          onPress={() => Linking.openURL(url)}
+          onPress={viewInExplorer}
```
Comment (LFBarreto) [inline src/screens/OperationDetails/Footer.js:49]: why track in the onPress callback when you already track this with the button click :thinking: 
I think we can move this to not do the event is already present

**Review by LFBarreto — CHANGES_REQUESTED** [review]

File: src/screens/Account/TokenContractAddress.js [hunk 2]
```diff
@@ -25,9 +26,13 @@ const forceInset = { bottom: "always" };
 
 const TokenContractAddress = ({ account, onClose, url, token }: Props) => {
   const { colors } = useTheme();
+
+  const currencyId = account ? account.token.name : token ? token.name : "";
+
   const viewInExplorer = useCallback(() => {
     if (url) {
       Linking.openURL(url);
+      track("ViewInExplorer", { currencyId });
```
Comment (LFBarreto) [inline src/screens/Account/TokenContractAddress.js:35]: Same issue here theres already an event on button click :shrug: `GoToViewContractInExplorer` same as the other one just add currencyId in eventProperties

**Review by LFBarreto — CHANGES_REQUESTED** [review]
:boom:

**Review by LFBarreto — APPROVED** [review]
:goat:

**Review by lruggiero-ledger — APPROVED** [review]

File: src/screens/Account/TokenContractAddress.js [hunk 3]
```diff
@@ -25,6 +25,9 @@ const forceInset = { bottom: "always" };
 
 const TokenContractAddress = ({ account, onClose, url, token }: Props) => {
   const { colors } = useTheme();
+
+  const currencyId = account ? account.token.name : token ? token.name : "";
```
Comment (JunichiSugiura) [inline src/screens/Account/TokenContractAddress.js:29]: This helper helps.

https://github.com/LedgerHQ/ledger-live-common/blob/2eb8c4f3bb8dc1adee636752750d2e8c01d06482/src/account/helpers.js#L54

**Review by JunichiSugiura — CHANGES_REQUESTED** [review]

File: src/screens/Account/TokenContractAddress.js (hunk 3 above)
Comment (valpinkman) [reply src/screens/Account/TokenContractAddress.js:29]: not relevant

**Review by JunichiSugiura — APPROVED** [review]

**Pull request merged**